CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - Move `buf mod {clear-cache,cc}` to `buf registry cc`. `buf mod {clear-cache,cc}` is now deprecated.
 - Deprecate `buf mod open`.
 - Delete `buf beta migrate-v1beta1`.
+- Promote `buf alpha sdk <go-version|maven-version|npm-version|swift-version|python-version>` to `buf registry sdk version`.
```

Comment:
Just say "Add `buf registry sdk version` to get the version of a Generated SDK for a module and plugin." or the like. As far as users are concerned, `alpha` doesn't exist, and we don't support it, so I wouldn't think of this as a promotion.